templates/web/smidsy/about/faq.html

```diff
@@ -3,95 +3,174 @@
-        <dt>Do you remove silly or illegal content?</dt>
-        <dd>Collideoscope are not responsible for the content and accuracy
-of material submitted by its users. We reserve the right to edit or remove any
-problems or updates which we consider to be inappropriate upon being informed
-by a user of the site.</dd>
-
-        <dt>Why does the site use kilometres for measurements?</dt>
-        <dd>The British national grid reference system, devised by Ordnance Survey (the British national
-        mapping agency) around the time of the second world war, uses eastings and
-        northings measured in metres and kilometres; the maps we use are from
-        Ordnance Survey and so this is what we use to display distances.
-        There you have it: not everything British is in miles!</dd>
-
-        <dt>Why can&rsquo;t I zoom out more on the reporting map?</dt>
-        <dd>        
-        We want to keep Collideoscope locally focused, so restrict the ability to
-        move radically between areas. The map on Your incidents will let you see all
-        the incidents you&rsquo;ve made, wherever they are. If you&rsquo;re from the
-        council then the emailed version of the problem incident also contains the
-        closest road and postcode to the pin on the map.</dd>
-
-        <dt>This site is great – why aren&rsquo;t you better publicised?</dt>
-        <dd>We're focussing on making Collideoscope as good as it can be for the moment, for the benefit of our users and our sponsors. As a result we rely on word of mouth to advertise the site.</dd> 
-    </dl>
-    
-    <h2><a name="organisation"></a>Organisation Questions</h2>
-    <dl>
-        <dt>Who built Collideoscope?</dt>
-        <dd>Collideoscope was built by <a href="http://www.mysociety.org/">mySociety</a> and <a href="http://itpworld.net">Integrated Transport Planning Ltd</a>. It is based upon the FixMyStreet project, <a href="http://www.fixmystreet.com/faq#organisation">See full credits for FixMyStreet</a>.
-        </dd>
-
-        <dt>Why build Collideoscope?</dt>
-        <dd>
-            <p>This website is our attempt to enable society to collaborate in order to collect some much-needed evidence to inform the ongoing cycle-safety debate. Our aim is to build a map of cycling-related incidents and near-misses for the UK so that transport planners, health researchers and decision-makers can see for the first time where incidents are happening, how often they take place, and how serious they are. We believe this kind of evidence fills a gap that will potentially enable local Councils to make proactive changes to highway environments so as to prevent serious, and potentially fatal, cycling incidents from occurring in the future.</p>
-        </dd>
-    </dl>
+
+<h2 name="general">General</h2>
```

Comment:
Think you mean `id`.

Reply:
Ah yes, sorry, I dropped the `<a>` tags, but forgot to swap to `id`s instead. Thanks for spotting! :+1: